File: templates/Virtual-Assistant-Template/typescript/samples/sample-assistant/src/services/botSettings.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */
import { ISkillManifest } from 'botbuilder-skills';
import { IBotSettingsBase } from 'botbuilder-solutions';

export interface IBotSettings extends IBotSettingsBase {
    skills: ISkillManifest[];
}

File: skills/csharp/experimental/weatherskill/Services/BotSettings.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using System.Collections.Generic;
using Microsoft.Bot.Builder.Skills.Models.Manifest;
using Microsoft.Bot.Solutions;

namespace WeatherSkill.Services
{
    public class BotSettings : BotSettingsBase
    {
        public string WeatherApiKey { get; set; }
    }
}